external/dir.props

```diff
@@ -8,5 +8,8 @@
     <RuntimeOS Condition="'$(PortableBuild)' == 'true' and '$(OSGroup)' == 'Linux'">linux</RuntimeOS>
     -->
     <NugetRuntimeIdentifier>$(RuntimeOS)-$(ArchGroup)</NugetRuntimeIdentifier>
+    <ContainsPackageReferences>true</ContainsPackageReferences>
+    <!-- We need configuration-specific assets files. -->
+    <RestoreOutputPath>$(BaseIntermediateOutputPath)external/$(MSBuildProjectName)/$(TargetGroup)</RestoreOutputPath>
```

Comment:
Better yet just hook it off IntermedateOutputPath (https://github.com/dotnet/corefx/blob/master/dir.props#L237)